website/docs/r/servicebus_namespace_authorization_rule.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_servicebus_namespace_authorization_rule"
+sidebar_current: "docs-azurerm-resource-servicebus-namespace-authorization-rule"
+description: |-
+  Manages a ServiceBus Namespace authorization Rule within a ServiceBus.
+---
+
+# azurerm_servicebus_namespace_authorization_rule
+
+Manages a ServiceBus Namespace authorization Rule within a ServiceBus.
+
+## Example Usage
+
+```hcl
+
+resource "azurerm_resource_group" "example" {
+  name     = "terraform-servicebus"
+  location = "West US"
+}
+
+resource "azurerm_servicebus_namespace" "example" {
+  name                = "${var.servicebus_name}"
+  location            = "${var.location}"
```

Comment:
neither of these variables exists in this context - can we hard-code example values here? e.g. `myservicebusnamespace` / `${azurerm_resource_group.example.location}` respectively